pkgs/development/tools/wasm-pack/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkgconfig, openssl }:
+
+rustPlatform.buildRustPackage rec {
+  name = "wasm-pack-${version}";
+  version = "v0.8.0";
```

Comment:
```suggestion
  version = "0.8.0";
```

Reply:
hmm, looks like github got rid of my other review part, which was changing `rev` below from `${version}` to `v${version}`

Reply:
Hi @tadeokondrak, thanks for reviewing this.

I've made the suggested change. Will push the change once I'm doing building and testing.